src/components/Blogs/Blogs.jsx

```diff
@@ -76,12 +105,14 @@ const Blogs = () => {
             <BlogsContainer>
                 <BlogsSection>
                     <RouterNavCreateButtonLink to={"/dashboard/blogs/create"}>Create Blog</RouterNavCreateButtonLink>
+                    <BlogFilter selectedBlogs={selectedBlogs} handleSelectedBlogs={handleSelectedBlogs} />
```

Comment:
Please, render this filter only if the user is registered, in my opinion, unregistered users don't follow others so this useless to show this filter to them.

Reply:
It's done